src/components/CheckBox.js

```diff
@@ -15,12 +15,19 @@ type Props = {
   style?: ViewStyleProp
 };
 
+const getAccessibilityLabel = (label: string, checked: boolean) =>
```

Comment:
Maybe we need and accessibility utils that would export these types of functions

Reply:
Agreed, accessibility module that exported `checkboxAccessibilityLabel(label: string, checked: boolean)` would be great. Then we can use it for the ListItem too.

Reply:
excuse my noob-ish-ness, but where would you put this?

Reply:
Have moved it out to the data folder. LMK if it's better somewhere else.